content/en/docs/setup/install/istioctl/index.md

```diff
@@ -83,6 +86,12 @@ to install the `demo` profile:
 $ istioctl install --set profile=demo
 {{< /text >}}
 
+## Check what's installed
+
+`Istioctl` saves a copy of the CR that was used to install Istio into the cluster, called `installed-state`. You can
+inspect this CR if you ever lose track of what is installed in a cluster. This CR is used to perform checks in some
+`istioctl` commands and should not be removed.
+
```

Comment:
Does it make sense to have to command to output the `installed-state` CR?

Reply:
Probably too trivial...and the page is already really long.